mne/time_frequency/psd.py

```diff
@@ -111,11 +101,11 @@ def _psd_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
     x_splits = np.array_split(x, n_jobs)
     f_psd = parallel(my_pwelch(d, noverlap=n_overlap, nfft=n_fft,
```

Comment:
is it really fft ? maybe power? or psd of windows.

there are many options. Just pick one that is not misleading